api/model/src/main/java/org/projectnessie/model/Validation.java

```diff
@@ -95,6 +95,24 @@ public final class Validation {
   public static final Pattern REF_NAME_PATH_ELEMENT_PATTERN =
       Pattern.compile(REF_NAME_PATH_ELEMENT_REGEX);
 
+  /*
+   * Default Cut-Off Policy can be a number or Duration or ISO instant.
```

Comment:
this comment may be applicable for `DEFAULT_CUT_OFF_POLICY_REGEX` not `DURATION_REGEX`